src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj

```diff
@@ -12,6 +12,7 @@
     <ProjectReference Include="..\Microsoft.PowerShell.ConsoleHost\Microsoft.PowerShell.ConsoleHost.csproj" />
     <ProjectReference Include="..\Microsoft.PowerShell.Security\Microsoft.PowerShell.Security.csproj" />
     <ProjectReference Include="..\System.Management.Automation\System.Management.Automation.csproj" />
+    <ProjectReference Include="..\Modules\PSGalleryModules.csproj" />
```

Comment:
When building nuget packages for individual assemblies, I think this reference will cause a nuget package to be generated for `PSGalleryModules.csproj` and the generated `Microsoft.PowerShell.SDK` package will have a reference to the `PSGalleryModules` nuget package. We should avoid creating the unneeded `PSGalleryModules` nuget package.

Reply:
Indeed, I'll remove this and do the restore on PSGalleryModules.csproj explicitly.